tests/ui/unicode-quote.rs

```diff
@@ -0,0 +1,20 @@
+// Provide a useful error message when attempting to escape a non-ascii quotation mark.
+// <https://github.com/rust-lang/rust/issues/128858>
+fn main() {
+    dbg!("\″"); //U+2033
+    //~^ERROR 4:12: 4:13: unknown character escape: `\u{2033}`
```

Comment:
```suggestion
    //~^ERROR unknown character escape: `\u{2033}`
```
you can drop out the line and column numbers here to make the error annotations more robust, the `.stderr` file also contains them so they're still tested